src/main/java/frc/trigon/robot/subsystems/arm/simulationarm/SimulationArmConstants.java

```diff
@@ -0,0 +1,87 @@
+package frc.trigon.robot.subsystems.arm.simulationarm;
+
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.controller.ElevatorFeedforward;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.wpilibj.simulation.ElevatorSim;
+import edu.wpi.first.wpilibj.simulation.SingleJointedArmSim;
+
+public class SimulationArmConstants {
+    private static final int
+            ANGLE_MOTOR_AMOUNT = 2,
+            ELEVATOR_MOTOR_AMOUNT = 2;
+    private static final DCMotor
+            ANGLE_MOTOR_GEARBOX = DCMotor.getNEO(ANGLE_MOTOR_AMOUNT),
+            ELEVATOR_MOTOR_GEARBOX = DCMotor.getNEO(ELEVATOR_MOTOR_AMOUNT);
+    private static final double
+            ANGLE_GEAR_RATIO = 12.4,
+            ELEVATOR_GEAR_RATIO = 23952853.3;
+    private static final double MINIMUM_ARM_LENGTH = 5;
+    private static final double ARM_LENGTH = 20;
```

Comment:
Rename this to `MAXIMUM_ARM_LENGTH` and remove the `private static final double`